src/content/release/breaking-changes/uiscenedelegate.md

```diff
@@ -0,0 +1,187 @@
+---
+title: UISceneDelegate adoption
+description: >
+  A guide for Flutter iOS developers to adopt Apple's UISceneDelegate protocol.
+---
+
+## Summary
+
+Apple is requiring iOS developers to adopt the UISceneDelegate protocol which
+changes the order of initialization when applications launch. Certain usages of
+Flutter inside `application:didFinishLaunchingWithOptions:` on iOS will have to
+be updated.
+
+## Background
+
+Most Flutter users will not have custom logic inside of
+`application:didFinishLaunchingWithOptions:`. Those apps will not need to do
+any migration.
```

Comment:
They will need to migrate their Info.plist if we can't automatically do it, right?